internal/backends/winit/Cargo.toml

```diff
@@ -20,10 +20,12 @@ path = "lib.rs"
 wayland = ["winit/wayland", "glutin/wayland", "copypasta/wayland"]
 x11 = ["winit/x11", "glutin/x11", "copypasta/x11"]
 renderer-femtovg = ["femtovg", "fontdb", "libc", "servo-fontconfig", "winapi", "dwrote", "imgref", "unicode-script", "ttf-parser", "rgb"]
-renderer-skia = ["skia-safe", "glow"]
+renderer-skia = ["skia-safe", "glow", "metal", "objc", "core-graphics-types", "foreign-types"]
```

Comment:
Does objc, metal, etc. need to be conditional on Mac? Or is it safe to have these on Linux/Windows?

Reply:
That is a way to add package dependencies based on a target arch, target family, etc. but not features. See also the last paragraph:

> Unlike in your Rust source code, you cannot use [target.'cfg(feature = "fancy-feature")'.dependencies] to add dependencies based on optional features. Use the [features] section instead: